thirdparty/kyaml/runfn/runfn_test.go

```diff
@@ -936,6 +936,38 @@ func TestCmd_Execute(t *testing.T) {
 	assert.Contains(t, string(b), "kind: StatefulSet")
 }
 
+func TestCmd_Execute_includeMetaResources(t *testing.T) {
+	dir := setupTest(t)
+	defer os.RemoveAll(dir)
+
+	// write a test filter to the directory of configuration
+	if !assert.NoError(t, ioutil.WriteFile(
+		filepath.Join(dir, "filter.yaml"), []byte(ValueReplacerYAMLData), 0600)) {
+		return
+	}
+
+	// write a Kptfile to the directory of configuration
+	if !assert.NoError(t, ioutil.WriteFile(
+		filepath.Join(dir, "Kptfile"), []byte("kind: Deployment"), 0600)) {
```

Comment:
there is a const for `Kptfile` now, we should use that probably.

Reply:
I found the test to be very unintuitive in this case where `Kptfile` contained a kind Deployment and it got changed to `StatefulSet` : ) [I think this will break eventually when we implement strict validations of Kptfile during `fn eval`]

I would recommend using kptfile as input and output here.

Another thing about test:
it will be good to do a test without `includeMetaResources` where kptfile is not modified.

Reply:
Updated the tests